### PR TITLE
Update meta.pug

### PR DESCRIPTION
**Pull request by neilrichler**
updated title and meta description

**What project are you opening a pull request for?**
- react-instantsearch (use v2 base)
- instantsearch.js (use develop base)

**Summary**

**Result**

### CODE REVIEW

File: docgen/layouts/common/meta.pug [hunk 1]
```diff
@@ -24,6 +24,6 @@ html
     meta(content='Boost search on your React apps with Algolia', property='og:description')
     meta(content='React InstantSearch', property='og:site_name')
 
-    title React InstantSearch | #{title}
+    title React InstantSearch | Lightning Fast React and React Native Search
```
Comment (bobylito) [inline docgen/layouts/common/meta.pug]: So we're clear here, this means that all the pages will have the same title. Is that what you want @neilrichler

Comment (vvo) [inline docgen/layouts/common/meta.pug]: Ah yep now probably not what we want, @neilrichler if you want that tagline only on the homepage, we can do that instead. Let us know.

Also the curren tagline on website homepage is "Lightning-fast search for React and React native apps" should we use that instead? In any case both should be consistent.

Comment (neilrichler) [inline docgen/layouts/common/meta.pug]: Got it - yes, let's only update it for the home page for now. I'm OK to use the current tagline, but we might want to adjust this in the future.

Comment (vvo) [inline docgen/layouts/common/meta.pug]: Gonna do it right now then, thanks

**Pull request closed without merging**